exercises/leap/.version

```diff
@@ -1 +1 @@
-2
\ No newline at end of file
+3
```

Comment:
I don't think this warrants a version bump.
None of the user facing test cases changed.